app/scripts/router.js

```diff
@@ -305,7 +309,10 @@ function (
 
       // Remove leading slashes
       var url = $(event.currentTarget).attr('href').replace(/^\//, '');
-
+      if (this.environment.isFramed() && url.indexOf('legal') > -1) {
+        self.window.open(url, '_blank');
```

Comment:
https://github.com/mozilla/fxa-content-server/pull/2643/files#r33779332 see, use both of `self` and `this` in the context. there is no nested function